README.md

```diff
@@ -75,6 +95,58 @@ To let the CodePush runtime know which deployment it should query for updates ag
 1. Open your app's `Info.plist` and add a new `CodePushDeploymentKey` entry, whose value is the key of the deployment you want to configure this app against (e.g. the Staging deployment for FooBar app). You can retreive this value by running `code-push deployment ls <appName>` in the CodePush CLI, and copying the value of the `Deployment Key` column which corresponds to the deployment you want to use. 
 2. In your app's `Info.plist` make sure your `CFBundleShortVersionString` value is a valid [semver](http://semver.org/) version (e.g. 1.0.0 not 1.0)
 
+## Plugin Configuration - Android
+
+After installing the plugin and sync-ing your Android Studio project with Gradle, you need to configure your app to consult CodePush for the location of your JS bundle, since it will "take control" of managing the current and all future versions. To do this, perform the following steps:
+
+1. Initialize the module in MainActivity.java:
+    
+    ```java
+    ...
+    // 1. Import the plugin class
+    import com.microsoft.codepush.react.CodePush;
+    
+    // 2. Optional: extend FragmentActivity if you intend to show a dialog prompting users about updates.
+    public class MainActivity extends FragmentActivity implements DefaultHardwareBackBtnHandler {
+        ...
+        
+        @Override
+        protected void onCreate(Bundle savedInstanceState) {
+            ...
+            // 3. Initialize CodePush with your deployment key and an instance of your MainActivity.
+            CodePush codePush = new CodePush("d73bf5d8-4fbd-4e55-a837-accd328a21ba", this);
+            ...
+            mReactInstanceManager = ReactInstanceManager.builder()
+                    .setApplication(getApplication())
+                    ...
+                    // 4. DELETE THIS LINE --> .setBundleAssetName("index.android.bundle")
+                    
+                    // 5. Let CodePush determine which location to load the most updated bundle from.
+                    // If there is no updated bundle from CodePush, the location will be the assets
+                    // folder with the name of the bundle passed in, e.g. index.android.bundle
+                    .setJSBundleFile(codePush.getBundleUrl("index.android.bundle"))
+                    
+                    // 6. Expose the CodePush module to JavaScript.
+                    .addPackage(codePush.getReactPackage())
+                    ...
+        }
+    }
+    ```
+
+2. Let the CodePush runtime know which deployment it should query for updates against. Be sure to set the `versionName` in your `android/app/build.gradle`:
```

Comment:
The deployment key was configured above. Should we change this to simply. "Ensure that the `versionName` property in your `android/app/build.gradle` file is set to a semver compliant value."